src/jobs/add-image-tag.yml

```diff
@@ -1,9 +1,14 @@
 description: >
   Install GCP CLI, if needed, and configure. Adds a tag to an existing image.
 
-executor: default
+executor: <<parameters.executor>>
```

Comment:
If we show this as:
```
executor:
  name: <<parameters.executor>>
```

The user can also supply the tag parameter to the executor or any other parameters we could set in the future.